### PR TITLE
Add LPA decorator to the API service

### PR DESCRIPTION
**Pull request by cooperaj**
Filters empty actors from the response from the API Gateway before passing on the LPA to  the caller.

### CODE REVIEW

File: service-api/app/src/App/src/Service/Lpa/LpaService.php [hunk 1]
```diff
@@ -54,7 +61,7 @@ public function __construct(
      */
     public function getByUid(string $uid) : ?Repository\Response\LpaInterface
     {
-        return $this->lpaRepository->get($uid);
+        return ($this->lpaFilterFactory)($this->lpaRepository->get($uid));
```
Comment (acsauk-mojd) [inline service-api/app/src/App/src/Service/Lpa/LpaService.php]: Nice - I don't tend to use `__invoke` very often but makes things succinct. I'm assuming the parentheses around the factory are required to make it invokable?

File: service-api/app/test/AppTest/Service/Lpa/LpaDataCleanseDecoratorTest.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                        'surname' => ''
+                    ]
+                ]
+            ]);
+
+        $decoratedLpaInterface = new LpaDataCleanseDecorator($lpaResponseProphecy->reveal());
+        $lpa = $decoratedLpaInterface->getData();
+
+        $this->assertCount(0, $lpa['attorneys']);
+    }
```
Comment (acsauk-mojd) [inline service-api/app/test/AppTest/Service/Lpa/LpaDataCleanseDecoratorTest.php]: Could potentially add a further two elements to the array or write a new test with attorneys that contain a first and/or last name to cover all paths.

File: service-api/app/src/App/src/Service/Lpa/LpaService.php (hunk 1 above)
Comment (cooperaj) [reply service-api/app/src/App/src/Service/Lpa/LpaService.php]: Correct

File: service-api/app/test/AppTest/Service/Lpa/LpaDataCleanseDecoratorTest.php (hunk 2 above)
Comment (cooperaj) [reply service-api/app/test/AppTest/Service/Lpa/LpaDataCleanseDecoratorTest.php]: I was literally just thinking that. Will do.

**Pull request merged**